src/xrpld/core/Config.h

```diff
@@ -80,6 +80,12 @@ struct FeeSetup
     /** The per-owned item reserve requirement in drops. */
     XRPAmount owner_reserve{2 * DROPS_PER_XRP};
 
+    /** The compute limit for Feature Extensions. */
+    std::uint32_t extension_compute_limit{0xffffffff};
```

Comment:
Please use decimal

Reply:
Fixed in ee76920b839f7e29aa749adfd22efea8d237e697